### PR TITLE
lab-site: Tweak lab contents and editor

### PR DESCRIPTION
**Pull request by juliaogris**
Tweak lab contents after review and feedback from @phy1um. Slightly update Evy
editor interface. Include a minor update to remove Chrome dev tool warning
around form elements requiring an `id` attribute. Remove autocorrect on iOS.
Make links in comments clickable.

I'm unsure if my hackery in the editor around making links clickable has
broken other things. Hope not.

### CODE REVIEW

File: frontend/module/highlight.js [hunk 1]
```diff
@@ -1,11 +1,20 @@
...
   return result
 }
 
+function tokenToSpan(token) {
+  if (token.type !== "comment") {
+    return `<span class="${token.type}">${escapeHTML(token.val)}</span>`
+  }
+  let words = escapeHTML(token.val).split(" ")
+  words = words.map((w) => (w.startsWith("https://") ? `<a href=${w} target="_blank">${w}</a>` : w))
+  console.log("comment words", words)
```
Comment (camh-) [inline frontend/module/highlight.js:14]: left over debugging?

File: frontend/play/index.js [hunk 2]
```diff
@@ -784,7 +784,8 @@ function clamp(val, min, max) {
 }
 
 function initEditor() {
-  editor = new Editor(".editor", { sessionKey: "evy-editor", highlighter: highlightEvy })
+  const options = { sessionKey: "evy-editor", highlighter: highlightEvy, id: "evy-editor" }
```
Comment (camh-) [inline frontend/play/index.js:787]: nit: this order of object fields is different to that of ai/nodex.js  - neither are alphabetical, and I'm not sure what the right order would be for an importance-based order.

File: frontend/module/highlight.js (hunk 1 above)
Comment (juliaogris) [reply frontend/module/highlight.js:14]: woops.

Comment (juliaogris) [reply frontend/module/highlight.js:14]: done.

File: frontend/play/index.js (hunk 2 above)
Comment (juliaogris) [reply frontend/play/index.js:787]: I'll make them alphabetical, but not sure what you mean with `ai/nodex.js`

Comment (juliaogris) [reply frontend/play/index.js:787]: nvm. found it.

**Review by camh- — APPROVED** [review]
:green_salad:

**Pull request merged**